apps/site/lib/site_web/views/helpers.ex

```diff
@@ -181,6 +181,7 @@ defmodule SiteWeb.ViewHelpers do
 
   @doc "Returns a css class: a string with hyphens."
   @spec route_to_class(Routes.Route.t()) :: String.t()
+  def route_to_class(nil), do: nil
```

Comment:
This should probably be `""` as the return, so existing code that expects it to always be a string will work (and so the typespec remains accurate).

Reply:
Done.